proto/zitadel/action/v2beta/action_service.proto

```diff
@@ -394,51 +471,123 @@ service ZITADELActions {
 }
 
 message CreateTargetRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
+  string name = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 1000},
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
+      example: "\"ip_allow_list\"";
+      min_length: 1
+      max_length: 1000
     }
   ];
-  Target target = 2 [
-    (validate.rules).message = {
-      required: true
+  // Defines the target type and how the response of the target is treated.
+  oneof target_type {
+    option (validate.required) = true;
+    RESTWebhook rest_webhook = 2;
+    RESTCall rest_call = 3;
+    RESTAsync rest_async = 4;
+  }
+  // Timeout defines the duration until ZITADEL cancels the execution.
+  // If the target doesn't respond before this timeout expires, then the connection is closed and the action fails.
+  google.protobuf.Duration timeout = 5 [
+    (validate.rules).duration = {gte: {}, lte: {seconds: 270}},
```

Comment:
also just noted now, what's the reason for allowing 4.5min timeouts?

Reply:
That was defined at one point, didn't change it, there is also an issue regarding the max timeout, which could have influence here https://github.com/zitadel/zitadel/issues/8426.
Should I add it there?